### PR TITLE
Ajc optimus debug

### PR DESCRIPTION
**Pull request by ambrosejcarr**
debugging some problems with the Optimus PR test. 

@jsotobroad Could you take a look at this and make sure everything checks out in the jenkins job? I had to make some adjustments there and in the docker configuration. 

https://gotc-jenkins.dsp-techops.broadinstitute.org/job/skylab-PR-test/

### CODE REVIEW

**Comment by ambrosejcarr** [comment]
jenkins retest

**Comment by ambrosejcarr** [comment]
git retest

**Comment by ambrosejcarr** [comment]
jenkins retest

**Comment by ambrosejcarr** [comment]
jenkins retest

**Comment by ambrosejcarr** [comment]
jenkins retest

**Comment by jsotobroad** [comment]
we should chat about this

**Comment by ambrosejcarr** [comment]
Sure; I'm out of meetings at 2p.

File: test/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+**Updating workflows that result in scientific changes:**
+In the event that a PR produces a desirable scientific change, the PR test will fail. In order to ensure that future PRs test against the correct "expected values", the test inputs must be updated to reflect the success state of the incoming pipeline. For optimus, this means updating the md5 hashes corresponding to the new output files, which are found in: 
+`test/optimus/pr/test_inputs.json` (PR)
+`test/optimus/scientific/test_inputs.json` (scientific)
+
+More complex tests may require different inputs, but the expected values should always be parameterized in the `test_inputs.json` files to facilitate easy updating. 
+
+**This branch currently fails PR tests**
```
Comment (jsotobroad) [inline test/README.md]: Branch specific information like this probably shouldnt be in the readme or the next section.  At least not presented as "in this PR"

**Pull request merged**